src/main/java/pl/bykowsi/kurs/tydzien1pd/service/Basket.java

```diff
@@ -0,0 +1,53 @@
+package pl.bykowsi.kurs.tydzien1pd.service;
+
+import org.apache.commons.math3.random.RandomGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import pl.bykowsi.kurs.tydzien1pd.model.Product;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class Basket {
+
+
+    private static final Integer quantityOfProducts = 5;
+    private final RandomGenerator randomGenerator;
+    private final int MaxPrice;
+    private final int MinPrice;
+
+    @Autowired
+    private Basket(RandomGenerator randomGenerator, @Value("${price.MaxPrice}") int maxPrice, @Value("${price.MinPrice}") int minPrice) {
+        this.randomGenerator = randomGenerator;
+        this.MaxPrice = maxPrice;
+        this.MinPrice = minPrice;
+    }
+
+    public List<Product> createAndGetRandomBasket() {
+        List<Product> defaultList = createListOfDifferentProducts();
```

Comment:
nazwy zmiennych można zastąpić `var`, jeśli masz java 11, np: ` var defaultList = createListOfDifferentProducts();`

Reply:
hmmm... ostatnio słyszałem opinie na temat varu, że jest to fajnie uproszczenie ale nie wiemy co się kryje pod tą zmienną i że oznaczenie typu zmiennej zawsze jest czytelniejsze niż sama nazwa. Ale nie upieram się przy tym